### PR TITLE
Common/movie page styles

### PR DESCRIPTION
**Pull request by Patryk0408**

### CODE REVIEW

File: src/common/Tile/styled.js
Comment (WerkaStrong) [inline src/common/Tile/styled.js:1]: In the rates are missing "/10" and the votes should be placed along the voting line:
image

**Review by WerkaStrong — APPROVED** [review]
I added some notes in the comments, check and correct and it will be ready for master ;)

File: src/features/Movie/Banner/styled.js [hunk 1]
```diff
@@ -12,9 +10,28 @@ export const ImageContainer = styled.div`
...
+	background-size: cover;
+`
+
+export const MovieImage = styled.img`
 	display: block;
 	width: 100%;
+	max-height: 770px;
+	margin: 0 auto;
+	max-width: 1368px;
 `
```
[the remark this answers is not in the document]
Comment (Patryk0408) [reply src/features/Movie/Banner/styled.js:35]: Added

File: src/features/Movie/Banner/index.js [hunk 2]
```diff
@@ -9,40 +9,39 @@ import {
...
 			<ImageContainer>
-				<Pleksa src={backdropUrl} />
+				<MovieImageWrapper>
+					<MovieImage src={backdropUrl} />
+					<Pleksa />
+				</MovieImageWrapper>
 				<Header>
 					<Title>{original_title}</Title>
 					<Rating>
 						<StarImage src={star} alt='Star'></StarImage>
```
[the remark this answers is not in the document]
Comment (Patryk0408) [reply src/features/Movie/Banner/index.js:42]: Repaired

File: src/features/Movie/Banner/index.js [hunk 3]
```diff
@@ -9,40 +9,39 @@ import {
...
 
-	const {
-		backdrop_path,
-		original_title,
-		vote_average,
-		vote_count,
-	  } = movieData;
+	const { backdrop_path, original_title, vote_average, vote_count } = movieData
+
+	const backdropUrl = backdrop_path ? `https://image.tmdb.org/t/p/original${backdrop_path}` : null
```
[the remark this answers is not in the document]
Comment (Patryk0408) [reply src/features/Movie/Banner/index.js:30]: Repaired. I added condition to show it only when we have background url

File: src/common/Tile/styled.js
[the remark this answers is not in the document]
Comment (Patryk0408) [reply src/common/Tile/styled.js:1]: repaired

**Pull request merged**